src/views/_screen-md-min.scss

```diff
@@ -1,4 +1,18 @@
 /*
-  WET-BOEW
-  @title: Medium view and over (screen only)
+	WET-BOEW
+	@title: Medium view and over (screen only)
  */
+ 
+.provisional {
+	&.gc-table {
+		.tbl-small-txt {
```

Comment:
What element would be needed for the **.tbl-small-txt** class to be used if it is a child of the `<table>` but a parent of the `<thead>`?

Reply:
This is a PR draft - I just brought in my original CSS and converted - I'm still working on it

Reply:
@GormFrank I do have basic documentation of how the classes would work - but be warned it is very outdated. I have to finish converting it all over and see what makes sense and what needs to be restructured. 

David E has also helped with the naming convention so some classes have changed name.

https://design.canada.ca/components/tables-alpha-demo.html

Reply:
I personnally very much like the look and semantics from what I can see in the example on design.canada.ca!